spec/models/user_spec.rb

```diff
@@ -92,4 +92,33 @@
       end
     end
   end
+
+  context 'permissions' do
+    context 'a new user' do
+      subject { create(:user, :confirmed) }
+
+      it 'can manage team members' do
+        expect(subject.can_manage_team?).to be_truthy
+      end
+
+      it 'can manage locations' do
+        expect(subject.can_manage_locations?).to be_truthy
+      end
+
+      context 'after reload' do
```

Comment:
Not sure if this test is worth it or not - was useful as I was switching away from the in-memory implementation, but might be excessive / misleading now.